.travis.yml

```diff
@@ -1,23 +1,46 @@
 language: android
 jdk: oraclejdk8
 sudo: false
+
 addons:
   apt:
     packages:
     - lynx
+
 android:
   components:
   - platform-tools
   - tools
   - build-tools-28.0.3
   - android-28
+  - android-22
   - extra-android-m2repository
-before_install: chmod +x gradle/buildWithTravis.sh
-script: gradle/buildWithTravis.sh
+  - sys-img-armeabi-v7a-android-22
+
+before_install:
+  - chmod +x gradle/buildWithTravis.sh
+
+before_script:
+  - android list targets
+  - echo no | android create avd --force -n test -t android-22 --abi armeabi-v7a
+  - emulator -avd test -no-audio -no-window &
+  - android-wait-for-emulator
+  - adb shell input keyevent 82 &
+
+script:
+  - ./gradlew clean build connectedCheck
+
+after_failure:
+  - lynx -dump app/build/outputs/gnag/gnag.html
+
+after_success:
+  - ./gradle/buildWithTravis.sh
```

Comment:
This is the script shouldn't be here. But I think it can be removed for now... It is not working in the moment